cmake/Modules/Findaom.cmake

```diff
@@ -0,0 +1,50 @@
+# - Try to find aom
+# Once done this will define
+#
+#  AOM_FOUND - system has aom
+#  AOM_INCLUDE_DIR - the aom include directory
+#  AOM_LIBRARIES - Link these to use aom
+#
+#=============================================================================
+#  Copyright (c) 2020 Andreas Schneider <asn@cryptomilk.org>
+#
+#  Distributed under the OSI-approved BSD License (the "License");
+#  see accompanying file Copyright.txt for details.
+#
+#  This software is distributed WITHOUT ANY WARRANTY; without even the
+#  implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#  See the License for more information.
+#=============================================================================
+#
+
+if (UNIX)
```

Comment:
PkgConfig is available on msys2 and other non-unix systems

Reply:
That's why it has `if (UNIX)` and not `if (UNIX AND NOT WIN32)`!

Reply:
msys2 cmake and windows cmake does not set UNIX to true

Reply:
On msys2's mingw-w64 cmake
```cmake
message("Unix: ${UNIX}")
message("mingw: ${MINGW}")
message("WIN32: ${WIN32}")
if (UNIX)
    message("Unix")
else()
    message("Not Unix")
endif()
```
```bash
cmake -P temp.cmake
Unix:
mingw:
WIN32: 1
Not Unix
```